core/dbt/cli/README.md

```diff
@@ -1 +1,49 @@
-TODO
+# Exception Handling
+
+## `requires.py`
+
+### `postflight`
+In the postflight decorator, the click command is invoked (i.e. `func(*args, **kwargs)`) and wrapped in a `try/except` block to handle any exceptions thrown.
```

Comment:
@jtcohen6 and/or @stu-k Since this represents a breaking change, do we have a ticket to get the docs site updated?  I'm specifically thinking of some integrators that use exit codes (specifically, Meltano although I'm sure there are more)

Reply:
Ahh.. ✅